src/ess/livedata/dashboard/plots.py

```diff
@@ -151,6 +151,34 @@ def _prepare_2d_image_data(data: sc.DataArray, use_log_scale: bool) -> sc.DataAr
             )
         return plot_data
 
+    @staticmethod
+    def _get_log_scale_clim(data: sc.DataArray) -> tuple[float, float] | None:
+        """
+        Compute clim for log scale, returning fallback if data is all NaN.
+
+        HoloViews' LogColorMapper fails when color_mapper.low is None (which
+        happens when all data is NaN). This provides explicit bounds to avoid
+        the "TypeError: '>' not supported between instances of 'NoneType' and 'int'"
+        error in _draw_colorbar.
+
+        Parameters
+        ----------
+        data:
+            Data array (after log scale masking, may contain NaN).
+
+        Returns
+        -------
+        :
+            Tuple of (low, high) bounds, or None if data has valid positive values.
+        """
+        vmin = float(data.data.nanmin().value)
+        vmax = float(data.data.nanmax().value)
+        # If all NaN, nanmin/nanmax return nan
+        if np.isnan(vmin) or np.isnan(vmax) or vmax <= vmin:
+            # Return placeholder bounds for empty/invalid data
+            return (1.0, 10.0)
+        return None
```

Comment:
So vmin and vmax are never returned?

Reply:
Yes, if not none we do not set the option and let Bokeh do the job.